tests/kola/Makefile

```diff
@@ -0,0 +1,13 @@
+DESTDIR ?=
+
+TESTDIRS := destructive nondestructive
```

Comment:
Could just use e.g. `find -type d` here, no?

Reply:
Mmm...yeah but we want to exclude `.`...I had to spend a bit thinking about a clean way to do that, but done.